android-auto-app/build.gradle.kts

```diff
@@ -49,7 +49,8 @@ dependencies {
 
   // Please review the compatibility guide. This app is showcasing the latest features.
   // https://github.com/mapbox/mapbox-maps-android/tree/main/extension-androidauto#compatibility-with-maps-sdk-v10
-  implementation("com.mapbox.maps:android:10.9.0-beta.2")
+//  implementation("com.mapbox.maps:android:10.9.0-beta.2")
+  implementation(project(":sdk"))
```

Comment:
we should not do this. we can adopt the new api downstream but we should not need to change dependency here

Reply:
but yeah it's necessary for testing, just flagging that we do not want to merge it

Reply:
sorry this comment is in the wrong place. moved it https://github.com/mapbox/mapbox-maps-android/pull/1782/files#r1009647628